From 965d74c2ed762f7d398c030383bcedf5c63eb776 Mon Sep 17 00:00:00 2001
From: elpaablo <donopwhile1@gmail.com>
Date: Sat, 27 Jun 2026 12:58:30 +0100
Subject: [PATCH] check_boot_jars: Whitelist more OPlus framework packages

Allows com.color.inner, android.os.oplusdevicepolicy packages on the bootclasspath to support the OPlus camera and gallery port.
---
 scripts/check_boot_jars/package_allowed_list.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/check_boot_jars/package_allowed_list.txt b/scripts/check_boot_jars/package_allowed_list.txt
index 22b3d5c35..0d917c0b2 100644
--- a/scripts/check_boot_jars/package_allowed_list.txt
+++ b/scripts/check_boot_jars/package_allowed_list.txt
@@ -301,6 +301,8 @@ oplus\..*
 vendor\.oplus\..*
 com\.oneplus\..*
 vendor\.oneplus\..*
+com\.color\.inner\..*
+android\.os\.oplusdevicepolicy\..*
 
 # QC adds
 com.qualcomm.qti
